src/routes/safe/components/Settings/SafeDetails/index.tsx

```diff
@@ -186,6 +187,14 @@ const SafeDetails = (): ReactElement => {
             </Block>
           )}
 
+          <Block className={classes.formContainer}>
+            <Heading tag="h2">Export your data</Heading>
+            <Paragraph>
+              Download your local storage data to keep your added safes and address book entries for later reimport.
```

Comment:
```suggestion
              Download your local data to keep your added Safes and address book entries for later.
```

Reply:
I went with Ivans suggestion as it also includes a reference to the new domain.

Reply:
I would still remove "storage" to be less technical and capitalise "safe", e.g. "Download your local data with your added Safes and address book. You can import it on..."